fly/fly.py

```diff
@@ -8,17 +9,14 @@
 class Fly:
     def __init__(
             self, concourse_url, executable='/usr/local/bin/fly',
```

Comment:
`/usr/local/bin/fly` should be a constant, eg `DEFAULT_FLY_BIN` and then have `executable=DEFAULT_FLY_BIN` or `executable=None` and `self.executable = executable or DEFAULT_FLY_BIN` later.